examples/kubernetes-kafka/kafka-sw-app.yaml

```diff
@@ -9,6 +8,7 @@ spec:
     metadata:
       labels:
         app: kafka
+        zgroup: kafkaTestApp
```

Comment:
why do we need to add the `zgroup` label?

Reply:
Mainly to make the test simpler, we can wait to all pods to be ready using this label.

Reply:
`kafkaTestApp` but where is this label being used?